gui/wxpython/iscatt/plots.py

```diff
@@ -513,30 +513,29 @@ def MergeImg(cats_order, scatts, styles, rend_dt, output_queue):
         else:
             MergeArrays(merged_img, rend_dt[cat_id]["dt"], styles[cat_id]["opacity"])
 
-        """
-            # c_img_a = np.memmap(
-            #     grass.tempfile(), dtype="uint16", mode="w+", shape=shape
-            # )
-            c_img_a = colored_cat.astype("uint16")[:, :, 3] * styles[cat_id]["opacity"]
-
-            # TODO apply strides and there will be no need for loop
-            # b = as_strided(
-            #     a,
-            #     strides=(0, a.strides[3], a.strides[3], a.strides[3]),
-            #     shape=(3, a.shape[0], a.shape[1]),
-            # )
-
-            for i in range(3):
-                merged_img[:, :, i] = (
-                    merged_img[:, :, i] * (255 - c_img_a)
-                    + colored_cat[:, :, i] * c_img_a
-                ) / 255
-            merged_img[:, :, 3] = (
-                merged_img[:, :, 3] * (255 - c_img_a) + 255 * c_img_a
-            ) / 255
-
-            del c_img_a
-        """
+            # # c_img_a = np.memmap(
+            # #     grass.tempfile(), dtype="uint16", mode="w+", shape=shape
+            # # )
+            # c_img_a = colored_cat.astype("uint16")[:, :, 3] * styles[cat_id]["opacity"]
```

Comment:
```suggestion
            # c_img_a = colored_cat.astype("uint16")[:,:, 3] * styles[cat_id]["opacity"]
```
Line (comment) was 1 char too long